networkx/convert_matrix.py

```diff
@@ -378,36 +378,36 @@ def from_numpy_matrix(A, parallel_edges=False, create_using=None):
     Simple integer weights on edges:
 
     >>> import numpy
-    >>> A=numpy.matrix([[1,1],[2,1]])
+    >>> A=numpy.matrix([[1, 1], [2, 1]])
     >>> G=nx.from_numpy_matrix(A)
 
-    If `create_using` is a multigraph and the matrix has only integer entries,
+    If ``create_using`` is a multigraph and the matrix has only integer entries,
     the entries will be interpreted as weighted edges joining the vertices
     (without creating parallel edges):
 
     >>> import numpy
     >>> A = numpy.matrix([[1, 1], [1, 2]])
-    >>> G = nx.from_numpy_matrix(A, create_using=nx.MultiGraph())
+    >>> G = nx.from_numpy_matrix(A, create_using = nx.MultiGraph())
```

Comment:
Don't add spaces around the equals sign when used to assign keyword arguments to a function.

http://legacy.python.org/dev/peps/pep-0008/#other-recommendations